VisionNode:
  ros__parameters:
    min_x: 485
    max_x: 780
    max_y: 360
    min_prob: 0.30

